### ISSUE TITLE
Stubbing an object/string to test Grunt tasks

### ISSUE THREAD

**Issue by thom4parisot** [sinonjs/sinon#318]
Hello,

thanks for Sinon, it definitely helped me testing Grunt tasks in a clean way. However, I'm encountering a new use case which caused me more troubles than expected.

At some point, it might be related to #184.

My use case is making sure a Grunt task will follow an expected behaviour according to the user CLI call.

So I basically do the following:

``` javascript
var sandbox = sinon.sandbox.create();
var taskCurrentStub = sandbox.stub(grunt.task.current, "target").returns("my-fake-grunt-target");
```

It will raise the following error:

```
TypeError: Object #<Object> has no method 'returns'
```

I've tried with the following, without better results:

``` javascript
var sandbox = sinon.sandbox.create();
var taskStub = sandbox.stub(grunt.task, "current").returns({ target: "my-fake-grunt-target" });
```

Basically, I can't change the existing API as I'm not its owner. 

The complete use case is living in this Gist, with the workaround I found to make that working.

> https://gist.github.com/oncletom/6267889/d994339b9086fb716ff02d2ae4abe63ef4e58791

Any hint to make that working, or maybe it's a bug somewhere?
I was very surprised by the behaviour as no error or warning is sent.

Thanks :-)
